### PR TITLE
Reject unused, unreserved 2019 CVEs from Rapid7

### PR DESCRIPTION
**Pull request by todb-r7**
Let me know if you'd prefer a different process for this, but I figure this is the easiest mechanism for MITRE folks.

### CODE REVIEW

File: 2019/5xxx/CVE-2019-5657.json [hunk 1]
```diff
@@ -1,18 +1,18 @@
 {
```
Comment (cve-team) [inline 2019/5xxx/CVE-2019-5657.json]: File name CVE ID and JSON CVE ID do not match.

File: 2019/5xxx/CVE-2019-5659.json [hunk 2]
```diff
@@ -1,18 +1,18 @@
 {
```
Comment (cve-team) [inline 2019/5xxx/CVE-2019-5659.json]: File name CVE ID and JSON CVE ID do not match.

File: 2019/5xxx/CVE-2019-5657.json (hunk 1 above)
Comment (todb-r7) [reply 2019/5xxx/CVE-2019-5657.json]: Dangit

**Comment by todb-r7** [comment]
Fixed the two stragglers. My grep-fu is weak!

**Pull request merged**